apng-drawable/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ android {
                 cppFlags += "-std=c++17"
                 cppFlags += "-fno-rtti"
                 cppFlags += "-fexceptions"
+                arguments += listOf("-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON")
```

Comment:
This is to support the 16KB page size in NDK27.
- https://developer.android.com/guide/practices/page-sizes#compile-16-kb-alignment